plotnine/scales/scale.py

```diff
@@ -521,25 +518,6 @@ def trans(self, value):
         self._trans = t
         self._trans.aesthetic = self.aesthetics[0]
 
-    @scale.limits.setter
-    def limits(self, value):
-        """
-        Limits for the continuous scale
-
-        Notes
-        -----
-        The limits are given in original dataspace
-        but they are stored in transformed space since
-        all computations happen on transformed data. The
-        labeling of the plot axis and the guides are in
-        the original dataspace.
-        """
-        limits = self.trans.transform(value)
-        try:
-            self._limits = np.sort(limits)
-        except TypeError:
-            self._limits = limits
-
```

Comment:
Okay, now that I recall, this code cannot be removed otherwise it will break setting the limits for transformed scales. I think the changes below should be enough.

And maybe this line,

```python
limits = self.trans.transform(value)
```

should deal with None e.g. `[100, None]`.

This should be 2 commits.

1. The case for `[100, None]`.
2. The breaks & labels.